lib/commons/dom/is-visible.js

```diff
@@ -226,6 +209,15 @@ function isVisible(el, screenReader, recursed) {
     return false;
   }
 
+  if (nodeName === 'summary') {
```

Comment:
What happens if there's more than one summary element?

Reply:
It's then partially ignored. By "partially" I mean it's hidden in closed `<details>`, but has some `<summary>`'s styling applied to it. This, however, is invalid HTML - there has to be exactly one `<summary>` element according to MDN.

<details>
<summary>Summary 1</summary>
<summary>Summary 2</summary>
Content
</details>